lib/resources/http.rb

```diff
@@ -25,13 +25,14 @@ class Http < Inspec.resource(1)
     "
 
     # rubocop:disable ParameterLists
-    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil)
+    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil, ssl_verify: true)
```

Comment:
Even if we are not using Faraday in future, I like the structure for the config `ssl: { verify: true }`. We already had a discussion about using `cert_path = "~/.chef/trusted_certs"` in the past. This `ssl` hash would allow us to easily add this support via `ssl: { verify: true , ca_path: "/usr/lib/ssl/certs" }`.